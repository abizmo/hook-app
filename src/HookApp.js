import React from 'react'

import CounterApp from './components/01-useState/CounterApp'
import CounterWithCustomHook from './components/01-useState/CounterWithCustomHook'
const HookApp = () => {
  return (
    <div>
      <CounterWithCustomHook />
      <CounterApp />
    </div>
  )
}

export default HookApp
