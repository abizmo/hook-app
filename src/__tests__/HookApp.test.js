import { shallow } from "enzyme"
import HookApp from "../HookApp"

describe('Hook App Tests', () => {
  test('should render and match to Snapshot', () => {
    const wrapper = shallow(<HookApp />)

    expect(wrapper).toMatchSnapshot();
  })
  
})
