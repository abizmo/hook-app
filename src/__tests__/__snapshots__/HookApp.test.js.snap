// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Hook App Tests should render and match to Snapshot 1`] = `
<Fragment>
  <button
    className="btn btn-secondary"
    onClick={[Function]}
  >
    ToggleTarea
  </button>
  <div>
    <CallbackHook />
    <Memorize />
    <Layout />
    <RealExampleRef />
    <FocusScreen />
    <MultiplesHooks />
    <FormWithCustomHook />
    <SimpleForm />
    <CounterWithCustomHook />
    <CounterApp />
  </div>
</Fragment>
`;
